inst/tests/test-tags.r

```diff
@@ -457,3 +457,43 @@ test_that("Non-tag objects can be coerced", {
   expect_identical(result2$singletons, "110d1f0ef6762db2c6863523a7c379a697b43ea3")
 
 })
+
+test_that("Latin1 and system encoding are converted to UTF-8", {
```

Comment:
Testing these things can be very tricky, since they may be OS/locale-dependent, e.g. on Linux,

```r
> Encoding("\377")
[1] "unknown"
```